Formula/e/energy.rb

```diff
@@ -0,0 +1,26 @@
+class Energy < Formula
+  desc "CLI is used to initialize the Energy development environment tools"
+  homepage "https://energye.github.io"
+  url "https://github.com/energye/energy/archive/refs/tags/v2.4.5.tar.gz"
+  sha256 "157beb93773637b75ffd0d72cc5ba3303d39065f6448b5272b5d9350da2eb47e"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    cd "cmd/energy" do
+      system "go", "build", *std_go_args(ldflags: "-s -w")
+    end
+  end
+
+  test do
+    output = shell_output("#{bin}/energy cli -v 2>&1")
+    assert_match "Current", output
+    assert_match "Latest", output
+
+    output = shell_output("#{bin}/energy env 2>&1")
+    assert_match "Get ENERGY Framework Development Environment", output
+    assert_match "GOROOT", output
+    assert_match "ENERGY_HOME", output
```

Comment:
```suggestion
    output = shell_output("#{bin}/energy cli -v")
    assert_match "Current", output
    assert_match "Latest", output

    output = shell_output("#{bin}/energy env")
    assert_match "Get ENERGY Framework Development Environment", output
    assert_match "GOROOT", output
    assert_match "ENERGY_HOME", output
```

Reply:
does your CLI has different version scheme?

```
$ energy cli -v
      GO
    ENERGY
$ CLI Current: 1.0.10
$ CLI Latest : 1.0.11
$ There new version available.
  Download: https://sourceforge.net/projects/energye/files/cli/energy-darwinarm-64.zip
```

Reply:
it should be 1.0.1 rather than 1.0.10?

Reply:
The title has been modified to 1.0.10
Thank you, CLI is indeed 1.0.10. I didn't notice that it would be related to the title